docs/spelling.txt

```diff
@@ -58,6 +58,8 @@ middleware
 multi
 multiplatform
 natively
+ncurses
```

Comment:
I think to have it in `ncursesw.rst` and `terminal.rst` is enough, no need to add it globally.